server/models/Collective.js

```diff
@@ -1587,12 +1600,22 @@ export default function(Sequelize, DataTypes) {
     })
       .then(oldMembers => {
         // remove the members that are not present anymore
-        const diff = difference(oldMembers.map(t => t.id), members.map(t => t.id));
+        const diff = differenceBy(
+          oldMembers.map(t => ({ id: t.id, MemberCollectiveId: t.MemberCollectiveId })),
```

Comment:
It `.map(t => ({ id: t.id, MemberCollectiveId: t.MemberCollectiveId }))` necessary? As we're extracting the `id` later with `const diffMemberIds = diff.map(m => m.id);` maybe this could work by doing simply `differenceBy(oldMembers, members, 'id')`